src/qnetvo/network_ansatz.py

```diff
@@ -40,18 +40,14 @@ class PrepareNode(NoiseNode):
     :param num_settings: The number of settings that the quantum function accepts.
     :type num_settings: int
 
-    :param static_settings: A set of fixed settings for the preparation node.
-    :type static_settings: optional, array-like
-
```

Comment:
Nice that we can get rid of static settings, and just specify directly with `requires_grad`!